### PR TITLE
doc improvements

### PR DESCRIPTION
**Pull request by garymm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,7 @@ _site
 .sass-cache
 .jekyll-metadata
 *.out
+
+# direnv, posh-direnv
```
Comment (faxu) [inline .gitignore:6]: What's the purpose of these additions?

**Review by faxu — COMMENTED** [review]
Can you stage this on your fork to view the html output? See https://github.com/microsoft/onnxruntime/wiki/Update-ONNX-Runtime-Docs#preview-change

File: .gitignore (hunk 1 above)
Comment (garymm) [reply .gitignore:6]: So people can use these tools, whose configs are easiest to keep in the repo but should not be checked in:
https://direnv.net/ or https://github.com/takekazuomi/posh-direnv/blob/master/README.rst

**Comment by garymm** [comment]
> Can you stage this on your fork to view the html output? See https://github.com/microsoft/onnxruntime/wiki/Update-ONNX-Runtime-Docs#preview-change

https://garymm.github.io/onnxruntime/docs/tutorials/tutorials/export-pytorch-model.html
(BTW, having tutorials/tutorials/ in the path seems wrong but I'll hold off on fixing that for now)

https://garymm.github.io/onnxruntime/docs/how-to/add-custom-op.html

https://garymm.github.io/onnxruntime/docs/how-to/build/inferencing.html

**Review by faxu — APPROVED** [review]

**Pull request merged**